crates/hyperqueue/src/server/state.rs

```diff
@@ -118,6 +120,9 @@ impl State {
                 submission_date: job.submission_date,
             },
         );
+        for (task_id, _) in job.tasks.iter() {
```

Comment:
Let's not pre-populate the map and make it sparse. In 99% of cases, there will be no entries in this map, so this would just waste memory. If an entry is not in the map, it means that there were no crashes.